lib/cloudError.ts

```diff
@@ -24,6 +24,10 @@ export interface CloudError extends Error {
    * @property {Array<CloudError>} [details] An array of CloudError objects specifying the details.
    */
   details?: Array<CloudError>;
+  /**
+   * @property {any} [innerError] The inner error parsed from the body of the http error response
+   */
+  innererror?: any;
```

Comment:
The doc has innerError (cameCase) and the property name is innererror (all lowercase). JSON object keys are case sensitive.

Reply:
This should follow camelCase convention `innerError`